zephyr/Kconfig

```diff
@@ -4,6 +4,23 @@ config MCUBOOT_CMAKELISTS_DIR
 	string "Path to the directory of the MCUBoot CMakeLists.txt file"
 	default "$MCUBOOT_BASE/boot/zephyr/"
 
+
+config MCUBOOT_USE_HEX_FILE
+	# Mandatory option when being built through 'zephyr_add_executable'
+	bool "Use hex file instead of building MCUBoot"
+
+if MCUBOOT_USE_HEX_FILE
+
+config MCUBOOT_HEX_FILE
+	# Mandatory option when being built through 'zephyr_add_executable'
+	string "MCUBoot hex file"
+
+endif # MCUBOOT_USE_HEX_FILE
+
+config MCUBOOT_SKIP_BUILD
```

Comment:
Could we express that there is a choice between skipping the build, building, and merging in a hex file?

With a Kconfig choice perhaps?

As it is now the user could both enable MCUBOOT_SKIP_BUILD and MCUBOOT_USE_HEX_FILE

Reply:
True, will try.

Reply:
Awesome :)